### PR TITLE
Patch for issue #105 and partial fix of issue #89

### PR DESCRIPTION
**Pull request by gsilvapt**
This stage creates another feature of the website, under "start", by providing a guide on what is Labs and what kind of activities we do here. A few things are missing in that regard, but can be improved later. Commit https://github.com/okfn/handbook/commit/a1d1ba82d30cd282b2d4cb454fd16b6c639a1c4d contains that information.

This fix also finalizes issue #90, providing a full guide on how to contribute to core datasets - https://github.com/okfn/handbook/commit/5b3d332b307dce90bf985048e70abb624fd098ce .

https://github.com/okfn/handbook/commit/fed530e201912322d49b3d5606700c350bbe5e79 **Fixes issue #105**, and partially 89 as mentioned. Forgot to add the 105 in the commit message.

In addition, a small fix was added to imrpove texts and similar: https://github.com/okfn/handbook/commit/9965e3133aa0acb8d139a12441758d470541af90

Lastly, the data wrangling walk-through (as per issue #105) was added in the data list of tutorials - as per commit https://github.com/okfn/handbook/pull/116/commits/ba7e00cef4ea79526c1b6e18d532b4f0cd3dfc32

This PR allows the site to be almost 100% functional, maybe needing some proofreading.

### CODE REVIEW

File: start/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Types of Activities
+
+### Data Wrangling
+
+Data wrangling can be described as the process of getting, cleaning and using data. The following list covers almost all activities we do.
+
+LIST
+
+Clicking [here]({{site.baseurl}}/data/tutorial), you can find a walk-through of the process of getting, cleaning and publishing data online. 
+#### Skills required
```
Comment (rufuspollock) [inline start/index.md]: should have a blank line above. I will let you fix these items in a separate PR.

File: data/index.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ The Handbook consists of two main parts:
...
 * [Patterns](patterns/): A set of "patterns" or recipes for doing specific tasks from scraping an HTML table to geocoding in a spreadsheet
 
-## Guides and Tutorials
+## Guides and Tutorials
 
 * [SQL for Data Manipulation](sql-for-data-manipulation/)
 * [Introduction CSV - the Lingua Franca of Data](csv/)
 * [Glossary of Terms](glossary/)
-
+* [Walk-through of some Data Wrangling basic tasks]({{site.baseurl}}/data/tutorial/)
```
Comment (rufuspollock) [inline data/index.md:21]: I'd put this as the top of the list :-)

**Comment by rufuspollock** [comment]
Generally excellent work.

One small comment on pull request best practice: it would be good to have one PR per area. e.g. final fixes for #90 would best be in a separate PR and work on #89 and and #105 would be separated.

Merging now ...

**Comment by gsilvapt** [comment]
Absolutely. Sorry for this one in particular. Once I started looking at each file I just kept rolling and completely forgot. I'll try to write down any non-related needs next time.

Thanks for the review. I'll upload a patch for these small things you suggested.

File: start/index.md (hunk 1 above)
Comment (gsilvapt) [inline start/index.md]: It already has the blank line between 19 and 20 there.

**Pull request merged**